core/fake/trillian_log_client.go

```diff
@@ -68,21 +68,26 @@ func (l *LogServer) GetLatestSignedLogRoot(ctx context.Context, in *tpb.GetLates
 }
 
 // GetSequencedLeafCount is not implemented.
-func (l *LogServer) GetSequencedLeafCount(ctx context.Context, in *tpb.GetSequencedLeafCountRequest, opts ...grpc.CallOption) (*tpb.GetSequencedLeafCountResponse, error) {
+func (*LogServer) GetSequencedLeafCount(context.Context, *tpb.GetSequencedLeafCountRequest, ...grpc.CallOption) (*tpb.GetSequencedLeafCountResponse, error) {
 	panic("not implemented")
```

Comment:
Unrelated to PR but should these return a canonical Unimplemented error rather than panic?

Reply:
Hmm.  Good point.  
I'm about to delete this in the next PR though.